README.md

```diff
@@ -23,6 +23,7 @@ SLACK_TOKEN={SLACK_TOKEN}
 SLACK_SIGNING_SECRET={SIGNING_SECRET}
 SLACK_CLIENT_SECRET={CLIENT_SECRET}
 SLACK_CLIENT_ID={CLIENT_ID}
+TRACKING_ID={TRACKING_ID}
```

Comment:
let's also include an entry in the task-definition.json for this:         

        {
          "valueFrom": "arn:aws:ssm:<aws-region>:<aws-account-id>:parameter/catalyst/<environment>/TRACKING_ID",
          "name": "TRACKING_ID"
        }

I'll add this to our dev and prod environments shortly.